react/features/invite/components/add-people-dialog/web/utils.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+import {
+    _decodeRoomURI,
+    getDialInfoPageURL,
+    shouldDisplayDialIn
+} from '../../../functions';
+
+/**
+ * Tries to copy a given text to the clipboard.
+ *
+ * @param {string} textToCopy - Text to be copied.
+ * @returns {boolean}
+ */
+export function copyText(textToCopy: string) {
+    const fakeTextArea = document.createElement('textarea');
+
+    // $FlowFixMe
+    document.body.appendChild(fakeTextArea);
```

Comment:
Won't this cause intermittent flashing of the layout? You seem to append a visible element to the body. I think it's wise to at least make sure with some CSS that it won't make layout changes, even for a very brief moment.

Reply:
Not at all from what I've tested, most likely due to the current layout.